<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>471</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>961</width>
    <height>471</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>961</width>
    <height>471</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Статистика магазина</string>
  </property>
  <widget class="QTableWidget" name="realizedFoodsStatsTable">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>461</width>
     <height>211</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Наименование</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Количество (шт)</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="realizedProductsStatsTable">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>461</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Наименование</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Масса (грамм)</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="globalStatsTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>461</width>
     <height>211</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Свойство</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Значение</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>Завершенных заказов:</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>Отмененных заказов:</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>Заказов в день (среднее)</string>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string>Средний чек:</string>
    </property>
   </item>
   <item row="4" column="0">
    <property name="text">
     <string>Заказов с доставкой:</string>
    </property>
   </item>
   <item row="5" column="0">
    <property name="text">
     <string>Заказов с самовывозом:</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ниже отражена глобальная статистика магазина:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ниже отражена статистика реализованных блюд:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ниже отражена статистика реализованных продуктов:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ниже отражена статистика бизнеса:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="moneyStatsTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>461</width>
     <height>151</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Свойство</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Значение (руб)</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>Себестоимость продуктов:</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>Прибыль от продаж:</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>Чистая прибыль:</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="closeStatsButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>971</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>-30</y>
     <width>20</width>
     <height>531</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
